docs/using_python_client_with_hazelcast_imdg.rst

```diff
@@ -2137,6 +2137,50 @@ See the following example.
     # Average age is 30
     print("Average age is %f" % average_age)
 
+Projections
+~~~~~~~~~~~~~~~~~
+
+Projections feature provides three projection functions; ``single_attribute``, 
+``multi_attribute`` and ``identity``, on top of Hazelcast ``Map`` entries. Their
+performance is high since they run in parallel for each partition and are 
+highly optimized for speed and low memory consumption.
+
+These projections are used with the ``map.project`` function, which takes an
+optional predicate argument.
+
+See the following example.
+
+.. code:: python
+
+    import hazelcast
+
+    from hazelcast.core import HazelcastJsonValue
+    from hazelcast.projection import single_attribute, multi_attribute
+    from hazelcast.predicate import greater_or_equal
+
+    client = hazelcast.HazelcastClient()
+    employees = client.get_map("employees").blocking()
+
+    employees.put(1, HazelcastJsonValue('{"Age": 23, "Height": 180, "Weight": 60}'))
+
+    employee_age = employees.project(single_attribute("Age"))
+    # Prints:
+    # The employee age is 23
+    print("The employee age is %d" % employee_age)
```

Comment:
Have you run this code? I believe the variable is a list, not a single objects. Please correct the output if it is something else.

```suggestion
    # The ages of employees are [23, 25]
    print("The ages of employees are %s" % employee_age)
```

Reply:
Same for others, please run the and verify they produce the output you wrote